account-kit/core/src/version.ts

```diff
@@ -1,3 +1,3 @@
 // This file is autogenerated by inject-version.ts. Any changes will be
 // overwritten on commit!
-export const VERSION = "4.66.0";
+export const VERSION = "4.66.1";
```

Comment:
these version changes should have already been done, as we're on version 4.66.1 already.